### PR TITLE
[Fix](explain) fix tablet showing problem

### PR DESCRIPTION
**Pull request by LiBinfeng-01**
tablets could not showed in explain string when selected partitions too large, so it need to be show in next line

### CODE REVIEW

**Comment by LiBinfeng-01** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java [hunk 1]
```diff
@@ -1352,7 +1352,7 @@ public String getNodeExplainString(String prefix, TExplainLevel detailLevel) {
         String selectedPartitions = getSelectedPartitionIds().stream().sorted()
                 .map(id -> olapTable.getPartition(id).getName())
                 .collect(Collectors.joining(","));
-        output.append(prefix).append(String.format("partitions=%s/%s (%s), tablets=%s/%s", selectedPartitionNum,
+        output.append(prefix).append(String.format("partitions=%s/%s (%s)\n tablets=%s/%s", selectedPartitionNum,
                 olapTable.getPartitions().size(), selectedPartitions, selectedTabletsNum, totalTabletsNum));
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java:1356]:
```suggestion
        output.append(prefix).append(String.format("partitions=%s/%s (%s)\n", selectedPartitionNum,
                olapTable.getPartitions().size(), selectedPartitions));
        output.append(prefix).append(String.format("tablets=%s/%s", selectedTabletsNum, totalTabletsNum));
```

Comment (LiBinfeng-01) [reply fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java]: done

**Comment by LiBinfeng-01** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java [hunk 2]
```diff
@@ -1352,8 +1352,9 @@ public String getNodeExplainString(String prefix, TExplainLevel detailLevel) {
         String selectedPartitions = getSelectedPartitionIds().stream().sorted()
                 .map(id -> olapTable.getPartition(id).getName())
                 .collect(Collectors.joining(","));
-        output.append(prefix).append(String.format("partitions=%s/%s (%s), tablets=%s/%s", selectedPartitionNum,
-                olapTable.getPartitions().size(), selectedPartitions, selectedTabletsNum, totalTabletsNum));
+        output.append(prefix).append(String.format("partitions=%s/%s (%s)", selectedPartitionNum,
+                olapTable.getPartitions().size(), selectedPartitions)).append("\n");
+        output.append(prefix).append(String.format("tablets=%s/%s", selectedTabletsNum, totalTabletsNum)).append("\n");
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java:1357]:
```suggestion
        output.append(prefix).append(String.format("tablets=%s/%s", selectedTabletsNum, totalTabletsNum));
```

**Comment by LiBinfeng-01** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by morrySnow** [comment]
run p0

**Review by starocean999 — APPROVED** [review]

**Comment by morrySnow** [comment]
run p0

**Pull request merged**